Carwash/orders/tests.py

```diff
@@ -1,3 +1,47 @@
-from django.test import TestCase
+from rest_framework.test import APITestCase
 
-# Create your tests here.
+from orders import factories
+from orders.models import Order
+
+
+class TestOrders(APITestCase):
+    """
+    Test CRUD operations on Order model.
+    """
+
+    def setUp(self) -> None:
+        """Create and save test instances with random values"""
+
+        self.account = factories.AccountFactory.create()
+        self.user = factories.UserFactory.create()
+        self.car_wash = factories.CarWashFactory.create()
+        self.order = factories.OrderFactory.create()
+        
+    def test_successful_create_data(self):
+        """Test if object is created and saved"""
+
+        data = Order.objects.all()
+        self.assertIsNotNone(data)
+    
+    def test_successful_delete_data(self):
```

Comment:
you don't send request to api endpoint

Reply:
Tests are added

Reply:
it's cool. but remove tests where you don't send request